docker/paddle_k8s

```diff
@@ -64,26 +64,24 @@ start_fluid_process() {
   trainer_label="paddle-job=${PADDLE_JOB_NAME}"
   task_index=""
 
-  if [ "${TRAINING_ROLE}" == "TRAINER" ] || [ "${TRAINING_ROLE}" == "PSERVER" ]; then
-    stdbuf -oL python /root/k8s_tools.py wait_pods_running ${pserver_label} ${PSERVERS}
+  if [ "${PADDLE_TRAINING_ROLE}" == "TRAINER" ] || [ "${PADDLE_TRAINING_ROLE}" == "PSERVER" ]; then
```

Comment:
I found our book demo https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/book/test_image_classification.py#L190 use `TRAINING_ROLE`, please keep them uniform, maybe you can create a new issue and we can fix them later.

Reply:
Thanks!